drivers/hmis/app/models/hmis/hud/custom_service.rb

```diff
@@ -19,7 +21,12 @@ class Hmis::Hud::CustomService < Hmis::Hud::Base
   belongs_to :user, **hmis_relation(:UserID, 'User'), inverse_of: :services, optional: true
   belongs_to :data_source, class_name: 'GrdaWarehouse::DataSource'
   belongs_to :custom_service_type
-  alias_attribute :service_type, :custom_service_type
+  def service_type = custom_service_type
```

Comment:
I've seen this syntax start to creep in, is this somethign we want to adopt for short methods?  It feel slightly awkward to me.

Reply:
It's the "endless method" syntax introduced in ruby 3. I guess I don't have a problem with it when used for simple accessors like this; it seems easy to read to me but that's subjective. If you want we could use alias_method instead which should be equiv in this case.

Reply:
I don't have a strong opinion, but it feels like a formatting decision we should make actively to keep the code consistent should all one line/one variable(?) methods be written that way?

Reply:
Maybe, I guess I would think of it as akin to using a line continuation: use it when it helps readability.